versioned_docs/version-8.7/apis-tools/camunda-api-rest/camunda-api-rest-overview.md

```diff
@@ -47,19 +43,30 @@ spring.servlet.multipart.max-request-size=4MB
 
 For example, if you increase the `maxMessageSize` to 10MB, increase these property values to 10MB as well.
 
+### Query API
+
+:::danger
+Query API endpoints do not currently support [resource authorizations][], and can be used to expand user access to restricted resources. If you use resource permissions, allowing public access to those endpoints is not recommended.
+:::
+
+All Query API endpoints contain an `(alpha)` declaration. Those endpoints are not accessible by default in Camunda 8 clusters.
+
+You can enable the [alpha feature][] search endpoints by setting either the configuration property `camunda.rest.query.enabled` to `true`,
+or the environment variable `CAMUNDA_REST_QUERY_ENABLED` to `true`.
+
 ## API Key Attributes
 
 OpenAPI key attributes have a `key` suffix, and they serve as the technical unique identifier for entities, such as
 `processDefinitionKey` and `tenantKey`, with `correlationKey` being the only exception. Those attributes are of type
-`string`. In previous versions, key attributes were defined as `integer (int64)`.
+`string` by default. In previous versions, key attributes were defined as `integer (int64)` by default.
```

Comment:
We can ditch these tiny edits, I felt they better explained the situation.

Reply:
👍